honeybee_radiance/modifier/material/bsdf.py

```diff
@@ -1,17 +1,15 @@
 """Radiance BSDF Material.
 
-http://radsite.lbl.gov/radiance/refer/ray.html#Glass
+https://floyd.lbl.gov/radiance/refer/ray.html#BSDF
 """
 from __future__ import division
 
 import os
-from .materialbase import Material
+from honeybee_radiance.modifier.material.absdf import aBSDF
```

Comment:
You should be able to use relative import here.

```suggestion
from .absdf import aBSDF
```